build/cpu.py

```diff
@@ -43,6 +43,11 @@ class IA64(CPU):
 	'''
 	name = 'ia64'
 
+class LOONGARCH64(CPU):
```

Comment:
```suggestion
class LoongArch64(CPU):
```
A web search suggests that "LoongArch" is the most common capitalization and it fits with the CamelCase typically used in Python class names.